js/currencycom.js

```diff
@@ -997,23 +1101,22 @@ module.exports = class currencycom extends Exchange {
                 throw new ArgumentsRequired (this.id + ' createOrder() requires an accountId parameter for ' + market['type'] + ' market ' + symbol);
             }
         }
-        const uppercaseType = type.toUpperCase ();
         const newOrderRespType = this.safeValue (this.options['newOrderRespType'], type, 'RESULT');
         const request = {
             'symbol': market['id'],
             'quantity': this.amountToPrecision (symbol, amount),
-            'type': uppercaseType,
-            'side': side.toUpperCase (),
+            'type': this.parseOrderType (type, true),
+            'side': this.parseOrderSide (side, true),
```

Comment:
Let's remove inverse mappings from this PR. It would be great if you could split it (one method per PR).

Reply:
yes, they were dependent on each other , they were all 'order' related and was easier to do them together.